tests/igc/test_reader.py

```diff
@@ -32,6 +32,28 @@ def test_decode_B_record():
     assert LowLevelReader.decode_B_record(line) == expected_result
 
 
+def test_process_B_record():
+
+    """Check whether correct extension information is taken from B record"""
+
+    # split up per extension for easy reading
+    i_record = 'I08' + '3638FXA' + '3941ENL' + '4246TAS' + '4751GSP' + '5254TRT' + '5559VAT' + '6063OAT' + '6467ACZ'
+    fix_record_extensions = LowLevelReader.decode_I_record(i_record)
+
+    assert {'bytes': (39, 41), 'extension_type': 'ENL'} in fix_record_extensions
+
+    # split up per 10 to enable easy counting
+    b_record = 'B093232520' + '2767N00554' + '786EA00128' '0019600600' '1145771529' + '3177005930' + '2770090'
+
+    expected_enl = int(b_record[38:41])  # because of pythons 0 indexing
+
+    decoded_b_record = LowLevelReader.decode_B_record(b_record)
+    processed_b_record = LowLevelReader.process_B_record(decoded_b_record, fix_record_extensions)
+
+    assert 'ENL' in processed_b_record
+    assert expected_enl == processed_b_record['ENL']
```

Comment:
we should be asserting against a constant here to make sure we don't screw up our own test logic here. if I read correct `00128` and `00196` are the two altitudes and after that are the extensions. the first extension is `FXA` with `006` and `ENL` is second with `001` which is `1` as an integer.

in other words, I think this should just be:
```python
assert processed_b_record['ENL'] == 1
```